### ISSUE TITLE
Cannot read property 'email' of undefined and other problems

### ISSUE THREAD

**Issue by seantwalsh3** [kalabox/kalabox#461]
This all started after my mac froze one night earlier this week, up to that point kbox was working fine. Ever since I had to restart my mac kbox git has not worked without me manually setting my session.email and session.name. In addition, I'm having an issue with terminus which I thought was and maybe still is related to issue #432. Any help is greatly appreciated, here is the verbose output of kbox git status -- -v.

```
kbox git status -- -v
debug: ENGINE =>  Initializing engine.
debug: ENGINE =>  Engine initialized.
debug: DISCO => AppsRegistry dirs. 0=dbl-1
debug: CONFIG => App config loaded. appCidsRoot=/Users/sean/.../dbl-1/.cids, name=pantheon-appserver, srcRoot=/Users/sean/.../dbl-1, proxy=[port=80/tcp, default=true, port=443/tcp, default=true, secure=true], name=pantheon-mariadb, srcRoot=/Users/sean/.../dbl-1, name=pantheon-solr, srcRoot=/Users/sean/.../dbl-1, name=pantheon-redis, srcRoot=/Users/sean/.../dbl-1, appName=dbl-1, appPlugins=[kalabox-plugin-git, kalabox-plugin-rsync, kalabox-plugin-pantheon], appRegistry=/Users/sean/.kalabox/appRegistry.json, appRoot=/Users/sean/.../dbl-1, apps=[kalabox-app-backdrop@^0.9.0, kalabox-app-drupal@^0.9.0, kalabox-app-pantheon@^0.9.0], appsRoot=/Users/sean/kalabox/apps, codeDir=code, codeRoot=/Users/sean/.../dbl-1/code, configSources=[DEFAULT_GLOBAL_CONFIG, ENV_CONFIG], domain=kbox, downloadsRoot=/Users/sean/.kalabox/downloads, engine=kalabox-engine-docker@^0.9.0, engineRepo=kalabox, globalPluginRoot=/Users/sean/kalabox/plugins, globalPlugins=[kalabox-core, kalabox-syncthing, kalabox-app-drupal, kalabox-app-backdrop, kalabox-app-pantheon], home=/Users/sean, kalaboxRoot=/Users/sean/kalabox, kboxRoot=/Users/sean/kalabox, logLevel=debug, logLevelConsole=info, logRoot=/Users/sean/.kalabox/logs, type=Darwin, platform=darwin, release=14.5.0, arch=x64, action=pull, drush-version=6, env=dev, framework=drupal, php=5.3.29, site=dbl1, url=git://github.com/pantheon-systems/drops-7.git, product_id=21e1fada-199c-492b-97bd-0b36b53a9da0, branch=master, profile=standard, provisioned=true, services=kalabox-services-kalabox@^0.9.0, shareIgnores=[*.7z, *.bz2, *.dmg, *.gz, *.iso, *.jar, *.rar, *.tar, *.tgz, *.un~, *.zip, .*.swp, .*DS_Store, .DS_Store*, ._*, .git, .sass-cache, Icon, Thumbs.db, ehthumbs.db, files, sites/default/files/*], sharing=true, srcRoot=/usr/local/lib/node_modules/kalabox, report=true, url=http://stats.kalabox.biz, sysConfRoot=/Users/sean/.kalabox, sysProviderRoot=/Users/sean/.kalabox/.provider, version=0.9.18
debug: DOCKER =>  Initializing provider.
debug: DOCKER =>  Provider initialized.
debug: EVENTS =>  Emitting event [dep-registered]. key=app, name=dbl-1, domain=dbl-1.kbox, url=http://dbl-1.kbox, dataContainerName=kb_dbl-1_data, root=/Users/sean/.../dbl-1, rootBind=/Users/sean/.../dbl-1, appCidsRoot=/Users/sean/.../dbl-1/.cids, appComponents=[name=pantheon-appserver, srcRoot=/Users/sean/.../dbl-1, proxy=[port=80/tcp, default=true, port=443/tcp, default=true, secure=true], name=appserver, hostname=appserver.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://appserver.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_appserver, containerIdFile=/Users/sean/.../dbl-1/.cids/appserver, containerId=aaf0b368f2decd303ddd2baa9e1e445f230812ddb1ff957ff6fba77c0c29bb34, name=pantheon-mariadb, srcRoot=/Users/sean/.../dbl-1, name=db, hostname=db.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://db.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_db, containerIdFile=/Users/sean/.../dbl-1/.cids/db, containerId=93fd3f33ff9ab109785d17319b461a5e63ca5b7ef952d055deff5c3bc11a4bf7, name=pantheon-solr, srcRoot=/Users/sean/.../dbl-1, name=solr, hostname=solr.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://solr.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_solr, containerIdFile=/Users/sean/.../dbl-1/.cids/solr, containerId=6b38b91198c61172afddd224479e3642d6be2ea1977917ad75631f6b3a3c77eb, name=pantheon-redis, srcRoot=/Users/sean/.../dbl-1, name=redis, hostname=redis.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://redis.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_redis, containerIdFile=/Users/sean/.../dbl-1/.cids/redis, containerId=d8bac5bbb5583b99c29c4f8d09d504157c5a707bacd2e0e7edbe40fbba789e5d], appName=dbl-1, appPlugins=[kalabox-plugin-git, kalabox-plugin-rsync, kalabox-plugin-pantheon], appRegistry=/Users/sean/.kalabox/appRegistry.json, appRoot=/Users/sean/.../dbl-1, apps=[kalabox-app-backdrop@^0.9.0, kalabox-app-drupal@^0.9.0, kalabox-app-pantheon@^0.9.0], appsRoot=/Users/sean/kalabox/apps, codeDir=code, codeRoot=/Users/sean/.../dbl-1/code, configSources=[DEFAULT_GLOBAL_CONFIG, ENV_CONFIG], domain=kbox, downloadsRoot=/Users/sean/.kalabox/downloads, engine=kalabox-engine-docker@^0.9.0, engineRepo=kalabox, globalPluginRoot=/Users/sean/kalabox/plugins, globalPlugins=[kalabox-core, kalabox-syncthing, kalabox-app-drupal, kalabox-app-backdrop, kalabox-app-pantheon], home=/Users/sean, kalaboxRoot=/Users/sean/kalabox, kboxRoot=/Users/sean/kalabox, logLevel=debug, logLevelConsole=info, logRoot=/Users/sean/.kalabox/logs, type=Darwin, platform=darwin, release=14.5.0, arch=x64, action=pull, drush-version=6, env=dev, framework=drupal, php=5.3.29, site=dbl1, url=git://github.com/pantheon-systems/drops-7.git, product_id=21e1fada-199c-492b-97bd-0b36b53a9da0, branch=master, profile=standard, provisioned=true, services=kalabox-services-kalabox@^0.9.0, shareIgnores=[*.7z, *.bz2, *.dmg, *.gz, *.iso, *.jar, *.rar, *.tar, *.tgz, *.un~, *.zip, .*.swp, .*DS_Store, .DS_Store*, ._*, .git, .sass-cache, Icon, Thumbs.db, ehthumbs.db, files, sites/default/files/*], sharing=true, srcRoot=/usr/local/lib/node_modules/kalabox, report=true, url=http://stats.kalabox.biz, sysConfRoot=/Users/sean/.kalabox, sysProviderRoot=/Users/sean/.kalabox/.provider, version=0.9.18, homeBind=/Users/sean, $ref=$["val"]["config"]["appComponents"]
debug: EVENTS =>  No listeners [dep-registered].
debug: EVENTS =>  Finished dispatching event listeners [dep-registered].
debug: EVENTS =>  Emitting event [app-registered-new]. name=dbl-1, domain=dbl-1.kbox, url=http://dbl-1.kbox, dataContainerName=kb_dbl-1_data, root=/Users/sean/.../dbl-1, rootBind=/Users/sean/.../dbl-1, appCidsRoot=/Users/sean/.../dbl-1/.cids, appComponents=[name=pantheon-appserver, srcRoot=/Users/sean/.../dbl-1, proxy=[port=80/tcp, default=true, port=443/tcp, default=true, secure=true], name=appserver, hostname=appserver.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://appserver.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_appserver, containerIdFile=/Users/sean/.../dbl-1/.cids/appserver, containerId=aaf0b368f2decd303ddd2baa9e1e445f230812ddb1ff957ff6fba77c0c29bb34, name=pantheon-mariadb, srcRoot=/Users/sean/.../dbl-1, name=db, hostname=db.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://db.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_db, containerIdFile=/Users/sean/.../dbl-1/.cids/db, containerId=93fd3f33ff9ab109785d17319b461a5e63ca5b7ef952d055deff5c3bc11a4bf7, name=pantheon-solr, srcRoot=/Users/sean/.../dbl-1, name=solr, hostname=solr.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://solr.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_solr, containerIdFile=/Users/sean/.../dbl-1/.cids/solr, containerId=6b38b91198c61172afddd224479e3642d6be2ea1977917ad75631f6b3a3c77eb, name=pantheon-redis, srcRoot=/Users/sean/.../dbl-1, name=redis, hostname=redis.dbl-1.kbox, appDomain=dbl-1.kbox, url=http://redis.dbl-1.kbox, dataContainerName=kb_dbl-1_data, containerName=kb_dbl-1_redis, containerIdFile=/Users/sean/.../dbl-1/.cids/redis, containerId=d8bac5bbb5583b99c29c4f8d09d504157c5a707bacd2e0e7edbe40fbba789e5d], appName=dbl-1, appPlugins=[kalabox-plugin-git, kalabox-plugin-rsync, kalabox-plugin-pantheon], appRegistry=/Users/sean/.kalabox/appRegistry.json, appRoot=/Users/sean/.../dbl-1, apps=[kalabox-app-backdrop@^0.9.0, kalabox-app-drupal@^0.9.0, kalabox-app-pantheon@^0.9.0], appsRoot=/Users/sean/kalabox/apps, codeDir=code, codeRoot=/Users/sean/.../dbl-1/code, configSources=[DEFAULT_GLOBAL_CONFIG, ENV_CONFIG], domain=kbox, downloadsRoot=/Users/sean/.kalabox/downloads, engine=kalabox-engine-docker@^0.9.0, engineRepo=kalabox, globalPluginRoot=/Users/sean/kalabox/plugins, globalPlugins=[kalabox-core, kalabox-syncthing, kalabox-app-drupal, kalabox-app-backdrop, kalabox-app-pantheon], home=/Users/sean, kalaboxRoot=/Users/sean/kalabox, kboxRoot=/Users/sean/kalabox, logLevel=debug, logLevelConsole=info, logRoot=/Users/sean/.kalabox/logs, type=Darwin, platform=darwin, release=14.5.0, arch=x64, action=pull, drush-version=6, env=dev, framework=drupal, php=5.3.29, site=dbl1, url=git://github.com/pantheon-systems/drops-7.git, product_id=21e1fada-199c-492b-97bd-0b36b53a9da0, branch=master, profile=standard, provisioned=true, services=kalabox-services-kalabox@^0.9.0, shareIgnores=[*.7z, *.bz2, *.dmg, *.gz, *.iso, *.jar, *.rar, *.tar, *.tgz, *.un~, *.zip, .*.swp, .*DS_Store, .DS_Store*, ._*, .git, .sass-cache, Icon, Thumbs.db, ehthumbs.db, files, sites/default/files/*], sharing=true, srcRoot=/usr/local/lib/node_modules/kalabox, report=true, url=http://stats.kalabox.biz, sysConfRoot=/Users/sean/.kalabox, sysProviderRoot=/Users/sean/.kalabox/.provider, version=0.9.18, homeBind=/Users/sean, $ref=$["app"]["config"]["appComponents"], $ref=$["app"]["config"]["appPlugins"]
debug: EVENTS =>  Start dispatching event listeners [app-registered-new].
debug: EVENTS =>  Finished dispatching event listeners [app-registered-new].
debug: EVENTS =>  Emitting event [pre-engine-create]. name=undefined, Binds=[], Env=[], NetworkMode=bridge, , VolumesFrom=[kb_dbl-1_data], WorkingDir=/code
debug: EVENTS =>  Start dispatching event listeners [pre-engine-create].
debug: EVENTS =>  Error dispatching event listeners [pre-engine-create]. Cannot read property 'email' of undefined
Cannot read property 'email' of undefined
TypeError: Cannot read property 'email' of undefined
    at getGitInfo (/Users/sean/.../dbl-1/plugins/kalabox-plugin-pantheon/lib/env.js:74:23)
    at /Users/sean/.../dbl-1/plugins/kalabox-plugin-pantheon/lib/env.js:244:21
    at /usr/local/lib/node_modules/kalabox/lib/core/events.js:147:11
    at /usr/local/lib/node_modules/kalabox/lib/core/events.js:146:24
    at /usr/local/lib/node_modules/kalabox/lib/core/events.js:130:23
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

**Comment by pirog** [comment]
So this issue should be fixed in the forthcoming 0.10.0 release. We are rolling that release next Friday aka Sept 18 2015. 

In the meantime I think you can workaround this issue by deleting `~/.terminus` and trying again.

**Comment by seantwalsh3** [comment]
Thanks @pirog got it working looking forward to the 0.10.0 release!
